lib/Dialect/Comb/CombOps.cpp

```diff
@@ -131,13 +131,26 @@ bool XorOp::isBinaryNot() {
 // ConcatOp
 //===----------------------------------------------------------------------===//
 
-void ConcatOp::build(OpBuilder &builder, OperationState &result,
-                     ValueRange inputs) {
+static unsigned valueRangeTotalWidth(ValueRange inputs)
```

Comment:
I'd recommend naming this `getTotalWidth`, there is no need to repeat "Value Range" as it is in the type info of the argument

Reply:
done